From a3d29e81ffea22c520356a7e967455492f7499f3 Mon Sep 17 00:00:00 2001
From: Mathieu Parent <math.parent@gmail.com>
Date: Fri, 13 Nov 2020 08:58:29 +0100
Subject: [PATCH] Install hook relative to $(DESTDIR)

---
 Makefile.am | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index fe9cd34..c6fc7bc 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -119,11 +119,11 @@ endif
 SUBDIRS = contrib
 
 install-exec-hook:
-	(cd $(ROOTSBINDIR) && ln -sf mount.cifs mount.smb3)
+	(cd $(DESTDIR)$(ROOTSBINDIR) && ln -sf mount.cifs mount.smb3)
 
 install-data-hook:
-	(cd $(man8dir) && ln -sf mount.cifs.8 mount.smb3.8)
+	(cd $(DESTDIR)$(man8dir) && ln -sf mount.cifs.8 mount.smb3.8)
 
 uninstall-hook:
-	(cd $(ROOTSBINDIR) && rm -f $(ROOTSBINDIR)/mount.smb3)
-	(cd $(man8dir) && rm -f $(man8dir)/mount.smb3.8)
+	(cd $(DESTDIR)$(ROOTSBINDIR) && rm -f $(DESTDIR)$(ROOTSBINDIR)/mount.smb3)
+	(cd $(DESTDIR)$(man8dir) && rm -f $(DEST)$(man8dir)/mount.smb3.8)
-- 
2.28.0

